pages/managing_relationships.md

```diff
@@ -321,4 +321,12 @@ Then, generate the `Citizen` entity:
     ? When you display this relationship with AngularJS, which field from 'Passport' do you want to use? id
 
 After doing this, a `Citizen` possesses a passport, but no `Citizen` instance is defined in `Passport`. On the generated AngularJS client UI you will have a dropdown in `Citizen` to select a `Passport` since `Citizen` is the owning side.
-This is not supported with JDL.
+This is the corresponding JDL:
+
+```
+entity Citizen
+entity Passport
+relationship OneToOne {
+  Citizen{passport} to Passport
```

Comment:
Yeah, for the relationship types allowed by JHipster

Le 5 avril 2016 16:14:54 GMT+02:00, Deepu K Sasidharan <notifications@github.com> a écrit :
>> @@ -321,4 +321,12 @@ Then, generate the `Citizen` entity:
>>      ? When you display this relationship with AngularJS, which field
>from 'Passport' do you want to use? id
>>  
>>  After doing this, a `Citizen` possesses a passport, but no `Citizen`
>instance is defined in `Passport`. On the generated AngularJS client UI
>you will have a dropdown in `Citizen` to select a `Passport` since
>`Citizen` is the owning side.
>> -This is not supported with JDL.
>> +This is the corresponding JDL:
>> +
>> +```
>> +entity Citizen
>> +entity Passport
>> +relationship OneToOne {
>> +  Citizen{passport} to Passport
>
>@MathieuAA so if I dont put the relationship name {} it wont be added
>as bidirectional is it?
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/jhipster/jhipster.github.io/pull/234/files/dbc89a7861fa3d110ceb3957439230e89b9cefc9#r58544973

-- 
Mathieu ABOU-AICHI